src/pages/public_registration/LogPanel.tsx

```diff
@@ -120,20 +126,58 @@ export const LogPanel = ({ tickets, registration }: LogPanelProps) => {
               </Typography>
             )}
           </Box>
-          <Typography>{new Date(registration.createdDate).toLocaleDateString()}</Typography>
+          <Tooltip title={new Date(registration.createdDate).toLocaleTimeString()} enterDelay={tooltipDelay}>
+            <Typography>{new Date(registration.createdDate).toLocaleDateString()}</Typography>
```

Comment:
Noe trøbbel med hvor tooltip dukker opp, ved at den ikke peker på elementet jeg peker på (teksten). En enkel fiks på det er sikkert å sette `width/height: "fit-content"` e.l. på `<Typography>`. Mulig det finnes en enda bedre/enklere måte også 🤔 
```suggestion
            <Typography  sx={{ height: 'fit-content' }}>{new Date(registration.createdDate).toLocaleDateString()}</Typography>
```

![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/7835b1be-1be5-469d-92de-dd4fee37e1ac)
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/a86f0774-bfcb-49ed-b89c-d42f11d19e4c)

Reply:
Godt spottet! Screenshot nr 2 prøvde jeg å få fikset, bl. a. med fit-content, men det gjorde mer skade enn nytte. Skal gjøre et nytt forsøk.

Reply:
Screenshot nr 1 fikset seg selv av å legge institusjon og bruker på egen rad